pupil-spa/src/app/services/check-complete/check-complete.service.spec.ts

```diff
@@ -0,0 +1,112 @@
+import { TestBed } from '@angular/core/testing';
```

Comment:
Non-blocking: I'd prefer this lengthy import list to be in alphabetical order.

Reply:
@jon-shipley done.